### PR TITLE
Add napari-ip-workflow recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@jaimergp, @jayunruh, do you agree with being added as maintainers?

**Comment by goanpeca** [comment]
@jayunruh, question: is the license `GPL-3.0-only` or `GPL-3.0-or-later? Thanks!

**Comment by jayunruh** [comment]
> https://conda-forge.org/#contribute

I'm happy to be listed as a maintainer.

**Comment by jayunruh** [comment]
> @jayunruh, question: is the license `GPL-3.0-only` or `GPL-3.0-or-later? Thanks!

I didn't know the "or-later" option was available but I'll go with that.  Do I need to change the included license file?

**Comment by goanpeca** [comment]
>I didn't know the "or-later" option was available but I'll go with that. Do I need to change the included license file?

Nope, all good :)

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/napari-ip-workflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  commands:
+    - pip list
+    # - pip check. # Fails due to use of 'skimage instead of 'scikit-image' in reqs
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jayunruh/napari-IP-workflow
+  summary: A plugin to do image preprocessing, segmentation, and measurements on other images.
+  license: GPL-3.0
```
Comment (jaimergp) [inline recipes/napari-ip-workflow/meta.yaml:41]:
```suggestion
  license: GPL-3.0-or-later
```

File: recipes/napari-ip-workflow/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - magicgui
+    - numpy
+    - pandas
+    - python >=3.8
+    - qtpy
+    - scikit-image
+
+test:
+  imports:
+    - napari_ip_workflow
```
Comment (jaimergp) [inline recipes/napari-ip-workflow/meta.yaml:31]:
```suggestion
    - napari_ip_workflow
    - napari_ip_workflow.ipwidgets
    - napari_ip_workflow.ipfunctions
    - napari_ip_workflow.segwidget
```

File: recipes/napari-ip-workflow/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - napari_ip_workflow.ipwidgets
```
Comment (jaimergp) [inline recipes/napari-ip-workflow/meta.yaml:32]:
```suggestion
```

**Pull request merged**